src/UI/Root/Components/Sidebar/Navigation/NavigationItem.tsx

```diff
@@ -113,9 +102,12 @@ const CompileReportItem: React.FC<Label & Url> = ({ label, url }) => {
         aria-label="Sidebar-Navigation-Item"
       >
         {label}
-        {data.kind === "Success" && data.value === true && (
+        {isSuccess && data === 200 && (
```

Comment:
why is data 200? and not the statuscode? Feels strange to me to assign the status code to the data object

Reply:
there is no status code in the react-query itself, there is FetchStatus but that's not the same, it's the status of the query, not status code of the request itself. the boolean value that was in the v1 request was the same assertion as it is here

Reply:
No, what I mean is, this useGetCompilerStatus query is normally a HEAD call, which doesn't return a response body. For me, it doesn't make sense that data should equal 200.